packages/flutter_test/lib/src/test_async_utils.dart

```diff
@@ -109,7 +109,7 @@ class TestAsyncUtils {
         throw FlutterError.fromParts(information);
       }
       if (error != null)
-        return Future<T>.error(error, stack);
+        return Future<T>.error(error! as Object, stack);
```

Comment:
Doesn't casting it to `Object` (as opposed to `Object?`) imply the `!`?

Reply:
Yes, it does. But @a14n is working on a lint that will require you to specifically think about null and add the `!` to say that you thought about nullability and this cannot be null here.

Reply:
OK. When that lint is turned on, I'm sure we'll have a bunch of places to fix.

Reply:
Yeah, a14n has already been fixing up our code base in prep for that.